# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../addresses/compile.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- Addresses-0.4.6/AddressManager/GNUmakefile.vanilla	2007-11-19 22:13:30.000000000 +0100
+++ Addresses-0.4.6/AddressManager/GNUmakefile	2007-11-19 22:14:07.000000000 +0100
@@ -23,8 +23,8 @@
 AddressManager_LANGUAGES=Dutch English French German Spanish
 AddressManager_OBJCFLAGS=-I../Frameworks -Wall
 AddressManager_LDFLAGS=\
-	-L../Frameworks/AddressView/AddressView.framework/Versions/A \
-	-L../Frameworks/Addresses/Addresses.framework/Versions/A     \
+	-L../Frameworks/AddressView/AddressView.framework/Versions/0 \
+	-L../Frameworks/Addresses/Addresses.framework/Versions/0     \
 	-lAddressView -lAddresses
 
 -include GNUmakefile.preamble
